bench.py

```diff
@@ -0,0 +1,84 @@
+import click
+import subprocess
+import os
+import sys
+import time
+import statistics
```

Comment:
since statistics is not in stdlib, PEP-8 recommends adding a blank line between stdlib imports and third-party imports